File: 88. Merge Sorted Array/Main.java
// https://leetcode.com/problems/merge-sorted-array/submissions/1072203904/
class Solution {
  public void merge(int[] nums1, int m, int[] nums2, int n) {
    ArrayList<Integer> aux = new ArrayList<>();
    for (int x = 0; x < m; x++)
      aux.add(nums1[x]);
    for (int x = 0; x < n; x++)
      aux.add(nums2[x]);
    Collections.sort(aux);
    n = 0;
    for (int num : aux) {
      nums1[n] = num;

      n++;
    }
  }
}

File: 88. Merge Sorted Array/main.cpp
/*
https://leetcode.com/problems/merge-sorted-array/submissions/1072203904/
*/
class Solution {
public:
    void merge(vector<int>& nums1, int m, vector<int>& nums2, int n) {
        vector<int>aux;
        for(int x=0; x<nums1.size()-n; x++)
        {
            aux.push_back(nums1[x]);
        }        
        for(int x=0; x<nums2.size(); x++)
        {
            aux.push_back(nums2[x]);
        }
        sort(aux.begin(), aux.end());
        for(int x=0; x<aux.size(); x++)
        {
            nums1[x] = aux[x];
            cout << nums1[x] << " ";
        }
    }
};